packages/drawnix/src/components/language-switcher/language-switcher.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useI18n, Language } from '../../i18n';
+import './language-switcher.scss';
+
+export const LanguageSwitcher: React.FC = () => {
```

Comment:
The component `LanguageSwitcher` is not be used, right?

If it is useless you could remove it.